libs/deps/mas/plans-modal.js

```diff
@@ -4232,6 +4232,7 @@ var setCustomTemplateLiteralTag = (e16) => {
 
 // ../node_modules/@spectrum-web-components/icons-ui/src/icons/Cross200.js
 var Cross200Icon = ({ width: t13 = 24, height: e16 = 24, title: r9 = "Cross200" } = {}) => tag`<svg
+    aria-hidden="true"
```

Comment:
The duplicates are still here, did you push your latest changes?

Reply:
I just found 1 more that was missed (L4462). Did you find more than that? 🤔

Reply:
Never mind, there are 2 plans-modal.js. One under `libs/deps/mas/` and one under `libs/deps/`. That's why I missed the duplicates you are talking about.